05_iigo/index.tex

```diff
@@ -1 +1,478 @@
+%\end{definition}
+
+
+\begin{definition} \label{def:tax}
+    The \textbf{taxation} is related to the President's \textbf{power} to request a specific \underline{\textbf{minimum}} amount of contribution from each island to the common pool at each round of the game. 
+\end{definition}
+
+\begin{definition} \label{def:alloc_req}
+    An \textbf{allocation request} is related to each island's \textbf{power} to request a specific amount of resource allocation from the President at each round of the game.
+\end{definition}
+
+
+\begin{definition} \label{def:rule_prop_list}
+A \textbf{rule proposal list} is related to each island's \textbf{power} to propose a specific rule to be passed to the President at each round of the game.
+\end{definition}
+
+\begin{definition} \label{def:invst}
+    An \textbf{investigation} is related to the Judge's \textbf{power} to acquire the information in making a decision followed by calculation of the expected results and checking for concurrency, exclusively for the actions carried out by the \textbf{islands}. 
+\end{definition}
+
+An example of an \emph{investigation}: The island $X$ has been permitted by the President to take the amount of $Y$ resources from the common pool. Upon an \emph{investigation} carried out by the Judge, it is revealed that the amount of resources taken out from the common pool by the island $X$ is in fact $Y'$ such that $Y' \neq Y$.
+
+
+\begin{definition}
+\textbf{Monitoring} is a government official's
+\textbf{power} to perform event recognition and checking for concurrency against the rules.
+\end{definition}
+
+An example of \emph{monitoring}: The Speaker has performed only the following action: \emph{counted the votes and calculated the result} for a rule. Upon a \emph{monitoring} carried out by the President, it is noticed that the Speaker has not made any \emph{announcement}. Hence, the Speaker has not followed their obligation to \emph{announce} the result of any vote held.
+
+See Section~\ref{sec:accountability} for more information about which roles can monitor which ones.
+
+
+\begin{definition}
+\textbf{Investigative-monitoring} is a government official's
+\textbf{power} to acquire the information used in acting on a governmental power followed by calculation of the expected results and checking for concurrency, exclusively for the actions carried out by a government official they are responsible for.
+\end{definition}
+
+An example of \emph{investigative-monitoring}: The Speaker has performed the following actions: \emph{counted the votes and calculated the result $R$} for a vote $V$ and \emph{announced} the result $R'$ for the vote $V$. Upon a \emph{investigative-monitoring} carried out by the President, it is noticed that $R' \neq R$. Hence, the Speaker has modified the announced result\footnote{Notice that \emph{investigative monitoring} involves tracking concurrency across actions.}.
```

Comment:
upon "an"

the style is consistent with the other definition -- I'm gonna keep it

Reply:
what if instead of changing the style but having issues with when an/a are used you remove the a - keeping the style the same but also making the sentence grammatically correct

e.g. Upon investigative-monitoring

Reply:
revised